libpod/config/default.go

```diff
@@ -58,7 +65,10 @@ func defaultConfigFromMemory() (*Config, error) {
 			"/usr/lib/cri-o-runc/sbin/runc",
 			"/run/current-system/sw/bin/runc",
 		},
-		// TODO - should we add "crun" defaults here as well?
+		"crun": {
+			"/usr/bin/crun",
+			"/usr/local/bin/crun",
+		},
```

Comment:
just wondering if we should increase the number of locations here to be similar to runc's list above it.  I'm mostly concerned about other platforms such as ubuntu.  @lsm5 any opinions?

Reply:
Good question. I copied those entries from the main `libpod.conf` so it should work fine on Fedora. I don't know where other distros put crun.

Reply:
Lets just put in the same paths for runc, for crun.

Reply:
Done. I also added them to the `libpod.conf`.